build.gradle

```diff
@@ -22,6 +22,7 @@ buildscript {
         ktlint_version = '0.50.0'
         gradle_download_task_version = '5.2.1'
         junit_version = '4.13.2'
+        mockk_version = '1.13.8'
```

Comment:
Is `mockito` recommended over `mockk` or should I be avoiding mocking libraries altogether?

Reply:
`mockito` is preferable and we've used it in AC without issues for many years now. There were useful methods in the `mockito-kotlin` library that were helpful as well but to avoid adding more dependencies, we [imported only those specific convenience functions](https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/support/test/src/main/java/mozilla/components/support/test/Mock.kt#1). 

If you can avoid a mocking library that would be ideal, but that is more of a philosophy than a good reason. 🙂